Automattic/jetpack
# Add recommendations initializer tweaks

## Proposed changes:
- **Add silent redBubble alerts - don't show in menu**
- **Hide recommendations when user purchases the product**

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Set up Jurassic Ninja site with Jetpack beta plugin
* Go to My Jetpack and go connect your site (first screen)
* When you see recommendations form, notice the red bubble in the nav menu with "1", reload the page
* You should still see recommendations form, but the red bubble in the navigation menu will be gone
* Submit the form, see the recommendations
* Purchase selected recommendation, when you're back to My Jetpack screen after checkout, they should be gone
